kii_thing_if.h

```diff
@@ -266,6 +266,21 @@ kii_bool_t init_kii_thing_if(
         kii_thing_if_state_updater_resource_t* state_updater_resource,
         KII_JSON_RESOURCE_CB resource_cb);
 
+/** Activate kii_thing_if_t instance.
+ *
+ * thing-if ThingSDK starts to receive command and update states with
+ * this function.
+ *
+ * This function must be called after one of following functions
+ * - ::onboard_with_vendor_thing_id,
+ * - ::onboard_with_thing_id
+ * - ::init_kii_thing_if_with_onboarded_thing
+ *
+ * @param [in] kii_thing_if_t This SDK instance.
+ * @return KII_TRUE when succeeded, KII_FALSE when failed.
+ */
+kii_bool_t activate(kii_thing_if_t* kii_thing_if);
```

Comment:
I prefer to call it 'start'.

Activate is good. I just felt start is more familiar word.